src/industry.h

```diff
@@ -138,12 +138,30 @@ struct Industry : IndustryPool::PoolItem<&_industry_pool> {
 		return IsTileType(tile, MP_INDUSTRY) && GetIndustryIndex(tile) == this->index;
 	}
 
+	/** Get produced cargo slot for a specific cargo type.
```

Comment:
`/**` usually goes on its own line

Reply:
Fixed, and fixed the existing comments in that file.